### PR TITLE
More info when child processes are killed

### PR DESCRIPTION
**Pull request by marcinzaremba**
- If child process reaches hard time limit executing a check report it's name (including interrupted check id)
- report PID of unexpectedly killed process

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: zmon_worker_monitor/workflow.py [hunk 1]
```diff
@@ -319,9 +314,9 @@ def process_message(queue, known_tasks, reactor, msg_obj, current_span, sampling
     if cur_time >= expire_time:
         current_span.set_tag(OPENTRACING_TASK_EXPIRATION, str(expire_time))
```
Comment (pitr) [inline zmon_worker_monitor/workflow.py]: would be good to add check id as the tag in current span

**Comment by marcinzaremba** [comment]
👍

**Comment by lfroment0** [comment]
:+1:

**Pull request merged**